songs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "adiherzog.github.io",
+    "version": "0.0.2",
+    "description": "",
+    "main": "index.html",
+    "scripts": {
+        "test": "eslint './**/*.js'"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/xeronimus/adiherzog.github.io.git"
```

Comment:
Yes I would change it anyway. Or will they be overwritten by npm later, e.g. when adding a new dependency? I don't think so, right?